bedrock/cms/management/commands/run_smartling_sync.py

```diff
@@ -32,3 +33,4 @@ def handle(self, *args, **kwargs):
                 sys.stdout.write("Snitch pinged\n")
         except Exception as ex:
             sys.stderr.write(f"\nsync_smartling did not execute successfully: {ex}\n")
+            capture_exception(ex)
```

Comment:
Making sure we get as much info as possible if the sync isn't happy for some reason